tests/e2e/gc_block_volume_expansion.go

```diff
@@ -2232,12 +2349,36 @@ func onlineVolumeResizeCheck(f *framework.Framework, client clientset.Interface,
 
 }
 
+// createStaticPVandPVCandPODinGuestCluster creates static PV and PVC in guest cluster.
+func createStaticPVandPVCandPODinGuestCluster(client clientset.Interface,
+	ctx context.Context, namespace string, svpvcName string, size string,
+	storageclass *storagev1.StorageClass, pvReclaimPolicy v1.PersistentVolumeReclaimPolicy) (*v1.PersistentVolumeClaim, *v1.PersistentVolume, *v1.Pod, string) {
+
+	gcPVC, gcPV := createStaticPVandPVCinGuestCluster(client, ctx, namespace, svpvcName, size, storageclass, v1.PersistentVolumeReclaimRetain)
```

Comment:
why are you hard coding the reclaim policy here

Reply:
This code is updated. This was not correct